plugins/auth-backend/package.json

```diff
@@ -83,25 +83,5 @@
     "dist",
     "migrations"
   ],
-  "configSchema": {
-    "$schema": "https://backstage.io/schema/config-v1",
-    "title": "@backstage/auth-backend",
-    "type": "object",
-    "properties": {
-      "auth": {
-        "type": "object",
-        "properties": {
-          "session": {
-            "type": "object",
-            "properties": {
-              "secret": {
-                "type": "string",
-                "visibility": "secret"
-              }
-            }
-          }
-        }
-      }
-    }
-  }
+  "configSchema": "config.d.ts"
```

Comment:
`"config.d.ts"` should be added to `"files"` here as well. :)
https://backstage.io/docs/conf/defining#schema-collection-and-definition

Reply:
My bad! updated as required, @OrkoHunter Thank you for pointing that out!